src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510056>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510057>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510058>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510060>))
```

Comment:
Move to env material hierarchy - is there no CHEBI role for colourant? That would help us with inks, dyes, etc.

The textual definition has a few issues:
>A manufactured product which contains a colourant that is used to imprint upon materials.

You don't need to use it to imprint on anything in order for it to be ink (intended use vs. realised role), maybe the capable of relation/wording is better to support later axiomatisation.

Perhaps: "An environmental material which contains at least one substance capable of absorbing light in the visible range"